lms/static/js/views/image_field.js

```diff
@@ -183,13 +184,13 @@
 
             showUploadInProgressMessage: function () {
                 this.$('.u-field-upload-button').css('opacity', 1);
-                this.$('.upload-button-icon').html(this.iconProgress);
+                this.$('.upload-button-icon').html(this.iconProgressHtml.toString());
```

Comment:
Does this mean that the toString method does some sort of escaping? I'm confused.